package.json

```diff
@@ -171,18 +171,18 @@
     "flow-bin": "^0.122.0",
```

Comment:
You can upgrade flow version to the one in the flow rules file: 0.137

Reply:
What is this file ?

Reply:
`.flowconfig`